tests/acceptance/features/bootstrap/bootstrap.php

```diff
@@ -0,0 +1,14 @@
+<?php
+$pathToCore = \getenv('PATH_TO_CORE');
+if ($pathToCore === false) {
+    $pathToCore = "../core";
+}
+
+require_once $pathToCore . '/tests/acceptance/features/bootstrap/bootstrap.php';
```

Comment:
Codacy found an issue: ["require_once" statement detected. File manipulations are discouraged. Concatenating is forbidden. Variables inside are insecure.](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=5898889)